app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,13 @@
+module LikesHelper
+  def liker(type, id, reaction)
```

Comment:
It is better to not put logic on helpers. Helpers are intended to simplify code that is used in the views, for example, format an string or use a gravatar for an email. Please, move this logic to another section in the project, like controllers or models.

Reply:
Actually on rails you put most of the logic on the helpers and if you read the article I'm adding, you will find our point of view. 
 
https://coderwall.com/p/jx9tca/when-i-use-helpers-partials-presenters-and-decorators?fbclid=IwAR2kUWZn8TbXxasOVuqB5XYJcWKK54IgbMtTLrkhn33mfHkxrpncY_iOHXo


If you have another information that touch this point please add it for us to check it.